addons/binding/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/KonnectedPutSettingsTimer.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.konnected.internal;
+
+import java.io.IOException;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
+
+import org.openhab.binding.konnected.handler.KonnectedHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class exists so that there is a delay while all of the
+ * items are linked to the channel and so that the put request
+ * does not go out until there has been a "calming" in the adding
+ * in the linking of the channels
+ *
+ *
+ * @author Zachary Christiansen - Initial contribution
+ */
+public class KonnectedPutSettingsTimer {
+    private Timer timer;
+    private final Logger logger = LoggerFactory.getLogger(KonnectedHandler.class);
+
+    public KonnectedPutSettingsTimer() {
+        this.timer = new Timer();
```

Comment:
Instead of creating your own timer you should use the `scheduler` inherited in your Thing handler, see D.2 in:

https://www.openhab.org/docs/developer/development/guidelines.html#d-runtime-behavior